addons/udes_stock/wizard/stock_backorder_confirmation.py

```diff
@@ -0,0 +1,22 @@
+from odoo import api, fields, models, _
+
+
+class StockBackorderConfirmation(models.TransientModel):
+    _inherit = "stock.backorder.confirmation"
+
+    u_disable_no_backorder_button = fields.Boolean(
+        string="Disable `No Backorder` button",
+        compute="_compute_disable_no_backorder_button",
+        help="Hide the `No Backorder` button if a user must always create a backorder when "
+        "validating a picking not fully available for that picking type.",
+    )
+
+    @api.depends("pick_ids")
+    def _compute_disable_no_backorder_button(self):
+        """Compute if the picking type must always create a backorder"""
+        Users = self.env["res.users"]
+        warehouse = Users.get_user_warehouse()
+        self.u_disable_no_backorder_button = (
+            self.pick_ids.picking_type_id
```

Comment:
This is assuming there is only one picking in `pick_ids`. Would be better to use `mapped`. We could even ensure we only have one picking type:
```
picking_type = self.mapped('pick_ids.picking_type_id')
picking_type.ensure_one()
```

Reply:
Fixed. Out of curiosity, how do you create a backorder for multiple pickings at once, I've only ever done it per picking in the UI.

Reply:
I don't know that, just raised it because it is is `pick_ids`.

Reply:
I'll leave it in though just so if they do find a way they can't break everything.